### DISCUSSION TITLE
85562E75-599C-45FB-AA91-8D28E47C9137

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#183] [General]
# 85562E75-599C-45FB-AA91-8D28E47C9137

The Nova-C spacecraft are a series of lunar landers developed by Intuitive Machines to deliver 100 kg of payload and instruments on the lunar surface. Part of NASA's Commercial Lunar Payload Services (CLPS) program alongside Peregrine, its first flight (IM-1) completed the first US lunar landing since Apollo 17 on 2024-Feb-22

https://celestia.mobi/addon?item=85562E75-599C-45FB-AA91-8D28E47C9137

**Comment by OlivaxNoel** [comment]
Looks cool
